README.md

```diff
@@ -1,40 +1,94 @@
-    xmlhttprequest: ~1.6.0 **NOTE**
-    socket.io-client: ~0.9.16 **NOTE**
+## Contents - DataSources
 
-**NOTE**: the final two listed dependencies are strictly speaking dev dependencies. However, when Kettle is loaded
-as a submodule for the purpose of running test cases, this fact cannot be propagated through the npm system. We must
-advertise them as 1st-class dependencies so that they can be available for 3rd party tests.
-    
+The Kettle repository also contains a few implementations of the simple `DataSource` contract for read/write access to data with a simple semantic (broadly the same as that
+encoded in [CRUD](https://en.wikipedia.org/wiki/Create,_read,_update_and_delete) although the current DataSource semantic does not provide explicitly for deletion). See the documentation section
+on [DataSources](./docs/DataSources.md) for details of this contract, the available implementations and how to use them.
 
-Kettle apps
--
+This repository contains DataSource implementations suitable for HTTP endpoints (with a particular variety specialised for accessing CouchDB databases with CRUDlike semantics) as well as the filesystem, with
+an emphasis on JSON payloads.
+
+# Getting Started and Community
+
+## Installation instructions
+
+Firstly, install node and npm by running a standard installer from [node.js](https://nodejs.org). Clone this repository and then run `npm install`.
+
+## Issue Tracking
+
+Issue tracking is at http://issues.fluidproject.org/browse/KETTLE .
+
+## IRC
+
+Visit `#fluid-work` on Freenode – community resources are linked at [Fluid's IRC Channels](https://wiki.fluidproject.org/display/fluid/IRC+Channel).
+
+## Mailing list
+
+Contact us on the [fluid-work](https://wiki.fluidproject.org/display/fluid/Mailing+Lists) mailing list with any problems or comments.
+
+## Uses of Kettle and related projects
+
+The primary user of Kettle is the [GPII](http://gpii.net/)'s autopersonalisation infrastructure, held at [GPII/universal](https://github.com/GPII/universal). Kettle is used
+to provide a flexible means of deploying the GPII's "Flow Manager" and related components distributed across multiple local and remote installations.
+
+A closely related project to Kettle is [gpii-express](https://github.com/GPII/gpii-express) which is used in other GPII projects such as the [terms registry](https://github.com/GPII/common-terms-registry) and
+[unified listing](https://github.com/GPII/ul-api). This is similar in architecture to Kettle (wrapping express primitives such as servers and requests into dynamically constructed Infusion components) 
+but slightly different in emphasis – 
+
+* gpii-express allows independently mounted application units with nested routing, in the Express 4.x style – whereas Kettle is currently limited to flat Express 3.x-style routing
+* Kettle incorporates support for WebSockets endpoints, whereas gpii-express does not
+* Kettle incorporates support for DataSources (see [DataSources](#datasources) )
```

Comment:
There are some links in the `README.md` that are referring to named anchors that have been moved to the `docs` directory:

- `(#datasources)` -> `(./docs/DataSources.md)`
- `(#testing)` -> `(./docs/KettleTestingFramework.md)`
- `(#docs)` -> `(./docs/)`